examples/v1alpha1/taskruns/build-push-kaniko.yaml

```diff
@@ -53,8 +53,6 @@ spec:
     - --destination=$(outputs.resources.builtImage.url)
     - --context=$(inputs.params.pathToContext)
     - --oci-layout-path=$(inputs.resources.builtImage.path)
-    securityContext:
```

Comment:
Not necessary as will run as root by default

Reply:
@danielhelfand this is not entirely true :stuck_out_tongue_closed_eyes: It depends on the kubernetes configuration. For example, we need this in OpenShift to make sure it knows it wants to run as root.

Reply:
Ah, yeah. Goes back downstream. My fault. Will add back.

Reply:
Added back